example/src/test/java/com/alipay/sofa/rpc/quickstart/QuickStartClient.java

```diff
@@ -32,9 +32,16 @@ public static void main(String[] args) {
 
         HelloService helloService = consumerConfig.refer();
 
+        //wait address to receive and connect successfully
```

Comment:
No need to sleep hear, you can use `consumerConfig.setConnectTimeout(10000)` (default is 3000) for waiting slow connection.